elasticsearch/Dockerfile.centos7

```diff
@@ -26,7 +26,7 @@ ENV ES_CLOUD_K8S_VER=2.4.4_01 \
 ARG ES_CLOUD_K8S_VER=2.4.4_01
 ARG OSE_ES_VER=2.4.4.20
 ARG ES_CLOUD_K8S_URL
-ARG OSE_ES_URL
+ARG OSE_ES_URL=https://github.com/jcantrill/openshift-elasticsearch-plugin/releases/download/2.4.4.21-snapshot/openshift-elasticsearch-plugin-2.4.4.21-SNAPSHOT.zip
```

Comment:
this will change once the plugin is released, correct?

Reply:
yes.  Thus the reason i made it do not merge